test/tools/jsontests/BlockChainTests.cpp

```diff
@@ -412,7 +412,6 @@ json_spirit::mObject fillBCTest(json_spirit::mObject const& _input)
 		}
 
 		blArray.push_back(blObj);  //json data
-		this_thread::sleep_for(chrono::seconds(1));
```

Comment:
Thread sanitizer reported that the `vptr` read and write are separated by this sleep.